src/main/java/com/jagrosh/jmusicbot/commands/dj/QueueTypeCmd.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018 John Grosh <john.a.grosh@gmail.com>.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.jmusicbot.commands.dj;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.commands.DJCommand;
+import com.jagrosh.jmusicbot.settings.QueueType;
+import com.jagrosh.jmusicbot.settings.Settings;
+
+/**
+ *
+ * @author Wolfgang Schwendtbauer
+ */
+public class QueueTypeCmd extends DJCommand
+{
+    public QueueTypeCmd(Bot bot)
+    {
+        super(bot);
+        this.name = "queuetype";
+        this.help = "changes the queue type";
+        this.arguments = "[" + String.join("|", QueueType.getNames()) + "]";
+        this.aliases = bot.getConfig().getAliases(this.name);
+        this.guildOnly = true;
+    }
+
+    @Override
+    protected void execute(CommandEvent event)
+    {
+        String args = event.getArgs();
+        QueueType value;
+        Settings settings = event.getClient().getSettingsFor(event.getGuild());
+
+        if (args.isEmpty())
+        {
+            QueueType currentType = settings.getQueueType();
+            event.reply("Current queue type is: " + currentType.getEmoji() + " `" + currentType.getUserFriendlyName() + "`.");
+            return;
+        }
+
+        try
+        {
+            value = QueueType.valueOf(args.toUpperCase());
+        }
+        catch (IllegalArgumentException e)
+        {
+            event.replyError("Invalid queue type. Valid types are: [" + String.join("|", QueueType.getNames()) + "]");
+            return;
+        }
+
+        if (settings.getQueueType() == value)
+        {
+            event.reply("The queue type is already set to `" + value.getUserFriendlyName() + "`.");
+            return;
+        }
```

Comment:
Seeing as a "is this already set" check doesn't exist with the other settings-related commands, I would remove this to stay consistent.